docs/reference/Image.rst

```diff
@@ -374,6 +374,10 @@ Constants
 ---------
 
 .. data:: NONE
+.. data:: MAX_IMAGE_PIXELS
+
+    Set to 89,478,485, approximately 0.25gb for a 24 bit (3 bpp) image.
+    See :py:meth:`~PIL.Image.open` for more information about how this is used.
```

Comment:
It's not actually used directly by `Image.open`, but by `_decompression_bomb_check` called by `_open_core` by `open`, so not so easy to find.

Let's combine the info from the [4.2.0](https://pillow.readthedocs.io/en/stable/releasenotes/4.2.0.html#new-decompressionbomb-warning) and [5.0.0](https://pillow.readthedocs.io/en/stable/releasenotes/5.0.0.html#decompression-bombs-now-raise-exceptions) release notes. 

Maybe something along these lines?

> Pillow emits warnings or errors for images that are unexpectedly large and may be a denial of service. 
>
> A ``DecompressionBombWarning`` is issued for images larger than the ``MAX_IMAGE_PIXELS`` threshold, and  a ``DecompressionBombError`` is raised for images that are twice as big.
>
> The default threshold can be changed or disabled by setting
``Image.MAX_IMAGE_PIXELS = None``.

Reply:
Ah, sorry, I misunderstood the first version!

Before, https://github.com/python-pillow/Pillow/commit/fa4d1493e26891b82f5f315c9afa683fdc8e0bd1 had:

```
See :py:meth:`~PIL.Image.open` for more information about how this is used.
```

... and was linking not to the `Image.open` _source_, but to [_documentation_](https://pillow--4802.org.readthedocs.build/en/4802/reference/Image.html#PIL.Image.open) which has this warning:

![image](https://user-images.githubusercontent.com/1324225/87942758-bb5bed80-caa5-11ea-9ee9-b877e617b19c.png)

In this warning, shall we mention and link to the constant?

```diff
-is over a certain limit.
+is over a certain limit, [MAX_IMAGE_PIXELS](link to the constant).
```

And let's also keep the link from the constant to `Image.open` docs.

And perhaps we should combine most of this info in a single place?

Reply:
Ok, I'm now linking to the constant. I'm confused though about how I can keep the link, while also trying to combine this info in one place?

Reply:
Yes, like this is good 👍

Shall we also mention `DecompressionBombError` in the first bit?